tasks/managed/prepare-fbc-snapshot/tests/test-prepare-fbc-snapshot-basic.yaml

```diff
@@ -58,14 +60,43 @@ spec:
               cat > "$(params.dataDir)/$(context.pipelineRun.uid)/data.json" << EOF
               {
                 "fbc": {
-                  "fromIndex": "registry-proxy.engineering.redhat.com/rh-osbs/iib-preview-rhtap:{{ OCP_VERSION }}",
-                  "targetIndex": "quay.io/redhat/redhat----preview-operator-index:{{ OCP_VERSION }}"
+                  "fromIndex": "registry-proxy.engineering.redhat.com/rh-osbs/iib-preview-rhtap:v4.12",
```

Comment:
shouldn't we make sure the replacement of `OCP_VERSION` was done successfully in this test?

Reply:
There is a separate test case for this. `test-prepare-snapshot-placeholder-replace.yaml`.